src/SparkFun_Si7021_Breakout_Library.cpp

```diff
@@ -60,15 +60,18 @@
   {
     Serial.println("Si7021 Found");
     //Serial.println(ID_Temp_Hum, HEX);
+    return true;
   }
   else if(x == 2)
   {
     Serial.println("HTU21D Found");
     //Serial.println(ID_Temp_Hum, HEX);
+    return true;
   }
   else
   	Serial.println("No Devices Detected");
   	//Serial.println(ID_Temp_Hum, HEX);
+	 return false;
```

Comment:
I think you need braces around these two indented statements.

Reply:
Ugh. Good catch.

Reply:
Well, while not pretty, it works as intended. If the IC is not x == 1 or 2, it drops and prints the 'No devices detected'. It then completes the if statement and returns false.

Said differently, if it's not a 0x15 or an 0x32, we don't know what it is and begin correctly returns false.